src/main/features/response/form/models/timelineBreakdown.ts

```diff
@@ -0,0 +1,66 @@
+import { ValidateNested } from 'class-validator'
+
+import { Serializable } from 'models/serializable'
+import { TimelineRow } from 'response/form/models/timelineRow'
+
+export const INIT_ROW_COUNT: number = 4
+
+export class TimelineBreakdown implements Serializable<TimelineBreakdown> {
```

Comment:
I think just `Timeline` would be enough. `TimelineBreakdown` sounds weird as well.

Reply:
I can also see many similarities with `ClaimAmountBreakdown` 🤔  Maybe we could extract some common parts as a subclass/utility class?

Reply:
because we have `AmountBreakdown` (that has very similar logic) I wanted to keep naming consistent.

Reply:
They are different though, I don't think the name makes sense

Reply:
ok

Reply:
ok